src/de/tuberlin/sese/swtpp/gameserver/model/Board.java

```diff
@@ -0,0 +1,85 @@
+package de.tuberlin.sese.swtpp.gameserver.model;
+
+public class Board {
+
+	protected int[][] board = new int[10][10];
+
+	// [0][0] ist das Feld oben links, also a9, [9][9] ist j0 um mit dem
+	// FenString konform zu sein.
+	// [0][1] ist demnach b9 und so weiter.
+	// 0 ist freies Feld, 1 ist Soldat weiss, 2 ist Stadt weiss, fuer Schwarz
+	// negative Zahlen.
+
+	public String getBoard() {
+		String result = "";
+		for (int i = 0; i < 10; i++) {
+			String tmp = "";
+			for (int j = 0; j < 10; j++) {
+				if (board[i][j] == 0)
+					tmp = tmp + "1";
```

Comment:
Wenn mehrere leere Zeilen aufeinander Folgen, wird nicht jedes mal ein `1` geschrieben, sondern die Zahlen addiert. Also es können dort alle reellen Zahlen stehen von `1 bis 9` oder auch `nichts`, die du in der `Zeile 30` schon beachtest hast.